authz/providers/azure/azure.go

```diff
@@ -48,26 +49,26 @@ type Authorizer struct {
 	rbacClient *rbac.AccessInfo
 }
 
-func New(opts authzOpts.Options, authopts auth.Options) (authz.Interface, error) {
+func New(opts authzOpts.Options, authopts auth.Options, dataActionsMap map[string][]map[string]map[string]oputil.DataAction) (authz.Interface, error) {
```

Comment:
should we declare a type for `map[string][]map[string]map[string]oputil.DataAction`? it's very hard to read